README.md

```diff
@@ -1,3 +1,14 @@
 # FixMe
 
-[![Build Status](https://github.com/CasBex/FixMe.jl/actions/workflows/CI.yml/badge.svg?branch=main)](https://github.com/CasBex/FixMe.jl/actions/workflows/CI.yml?query=branch%3Amain)
+[![Stable Documentation](https://img.shields.io/badge/docs-stable-blue.svg)](https://CasBex.github.io/FixMe.jl/stable)
+[![In development documentation](https://img.shields.io/badge/docs-dev-blue.svg)](https://CasBex.github.io/FixMe.jl/dev)
+[![Build Status](https://github.com/CasBex/FixMe.jl/workflows/Test/badge.svg)](https://github.com/CasBex/FixMe.jl/actions)
+[![Test workflow status](https://github.com/CasBex/FixMe.jl/actions/workflows/Test.yml/badge.svg?branch=main)](https://github.com/CasBex/FixMe.jl/actions/workflows/Test.yml?query=branch%3Amain)
+[![Lint workflow Status](https://github.com/CasBex/FixMe.jl/actions/workflows/Lint.yml/badge.svg?branch=main)](https://github.com/CasBex/FixMe.jl/actions/workflows/Lint.yml?query=branch%3Amain)
+[![Docs workflow Status](https://github.com/CasBex/FixMe.jl/actions/workflows/Docs.yml/badge.svg?branch=main)](https://github.com/CasBex/FixMe.jl/actions/workflows/Docs.yml?query=branch%3Amain)
+[![Coverage](https://codecov.io/gh/CasBex/FixMe.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/CasBex/FixMe.jl)
+[![DOI](https://zenodo.org/badge/DOI/FIXME)](https://doi.org/FIXME)
+
+## How to Cite
+
+If you use FixMe.jl in your work, please cite using the reference given in [CITATION.cff](https://github.com/CasBex/FixMe.jl/blob/main/CITATION.cff).
```

Comment:
```suggestion
```
This is a great default but not necessary here.

Reply:
I'll delete it.